.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can you please remove this file and add `.vscode` to `.gitignore`? Because this is very specific to your local development.

Reply:
Ohh! Ofc! Tyvm for catch that.

Reply:
I think it already there. Its not mine commit. If we rebase from gardens to gardens-staging it will dissapear, you know why?

Reply:
I'll remove it anyway and add to gitignore

Reply:
It because some weird rebase i think.

Reply:
I reverted the merges made here and rebase, now looks great. @reznord

Reply:
Removed and added on gitignore